contracts/ERC20Mintable.sol

```diff
@@ -0,0 +1,383 @@
+
+// File: openzeppelin-solidity/contracts/GSN/Context.sol
+
+pragma solidity ^0.6.6;
```

Comment:
is any of this "ported manually to solc 6" relevant anymore? https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Context.sol is for `^0.8.0` already...

is the mintable needed for a test only? Then the whole contract could be a lot shorter...